owncloudApp/src/main/res/values-ja-rJP/strings.xml

```diff
@@ -243,8 +243,7 @@
   <string name="auth_wtf_reenter_URL">予期せぬ状態；別のサーバーアドレスを入力してみてください</string>
   <string name="auth_expired_oauth_token_toast">認証情報は有効期限切れです。再度認証を行ってください。</string>
   <string name="auth_expired_basic_auth_toast">現在のパスワードを入力してください</string>
-  <string name="auth_expired_saml_sso_token_toast">セッションの有効期限切れです。再度接続してください。</string>
-  <string name="auth_connecting_auth_server">認証サーバーに接続中...</string>
+    <string name="auth_connecting_auth_server">認証サーバーに接続中...</string>
   <string name="auth_unsupported_auth_method">サーバーはこの認証方式をサポートしていません</string>
```

Comment:
Why is this line indented?

Reply:
Reason in https://github.com/owncloud/android/pull/2647#discussion_r319023746